visualizer/draw/area-chart.js

```diff
@@ -175,15 +236,49 @@ class AreaChart extends HtmlContent {
       this.draw()
     }
   }
+
   initializeFromData () {
     if (!this.initialized) {
       // These actions aren't redone when data changes, but must be done after data was set
       this.initialized = true
 
       // Same behaviour on mouse movements off coloured area as on
-      this.d3AreaChartSVG.on('mousemove', () => {
-        this.showSlice(d3.event)
-      })
+      this.d3AreaChartSVG
+        .on('mousemove', () => {
+          this.showSlice(d3.event)
+          const d = this.getHiddenMouseValues(d3.event)
+          if (!d) return
+          const layoutNodeId = extractLayoutNodeId(d.key)
+          if (!layoutNodeId || this.isInHoverBox) return
```

Comment:
In addition to returning here, can we have this behave like mouse-out:

1. Remove the current highlighting (`this.topmostUI.highlightNode(null)` IIRC)
2. It'd be nice to keep the old behaviour of using `cursor: pointer` to inform the user that something will happen when they click on an area chart area. Maybe set or unset a class or style on this condition (ideally without causing an additional reflow if possible?)